src/cli_commands/cli_status.c

```
@@ -7,6 +7,10 @@
 
 #include "cli_common.h"
 
+int print_nonsense() {
+  printf("This function does nothing, this is a testing PR\n");
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)